packages/gatsby-cli/src/create-cli.js

```diff
@@ -20,14 +22,14 @@ function buildLocalCommands(cli, isLocalSite) {
   const DEFAULT_BROWSERS =
     installedGatsbyVersion() === 1
       ? [`> 1%`, `last 2 versions`, `IE >= 9`]
-      : [`>0.25%`, `not dead`]
+      : [`> 0.25%`, `not dead`]
 
   let siteInfo = { directory, browserslist: DEFAULT_BROWSERS }
   const useYarn = existsSync(path.join(directory, `yarn.lock`))
   if (isLocalSite) {
     const json = require(path.join(directory, `package.json`))
     siteInfo.sitePackageJson = json
-    siteInfo.browserslist = json.browserslist || siteInfo.browserslist
+    siteInfo.browserslist = getBrowsersList(directory, DEFAULT_BROWSERS)
```

Comment:
This seems kind of weird to have the set up working like this, i.e. gatsby-cli _sets up_ some program stuff, and then other plugins use it. Might try to improve this.

Reply:
Yeah, can't remember why gatsby-cli does this